java/app/src/main/AndroidManifest.xml

```diff
@@ -1,29 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-
-    package="com.google.android.gms.samples.wallet">
+    package="com.google.android.gms.samples.wallet" >
 
     <application
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
-        android:supportsRtl="true">
+        android:supportsRtl="true" >
```

Comment:
TL;DR – Don't like it either, but let's keep it in order to follow conventions.
---
This is the result of a revision to a XHTML spec to ensure correct parsing of XHTML (the XML part) correctly by browsers. One could argue that manifests are ever going to be parsed as XHTML by browsers. What's more relevant, is that this leads to a default setting in Android studio, that is enabled by default, hence when "Code -> Reformatting code" this space is added unless changed otherwise.